lib/MusicBrainz/Server/Report/ReleasesMissingDiscIds.pm

```diff
@@ -0,0 +1,35 @@
+package MusicBrainz::Server::Report::ReleasesMissingDiscIds;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub query {
+    q{
+        SELECT DISTINCT ON (r.id)
+            r.id AS release_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM
+        ( SELECT r.id, r.artist_credit, r.name
+          FROM release r
+          WHERE id IN (
+              SELECT release FROM medium
+              WHERE (SELECT has_discids FROM medium_format WHERE medium_format.id = medium.format)
+              AND id NOT IN (SELECT medium FROM medium_cdtoc)
+          )
+        ) r
+    }
+}
+
+__PACKAGE__->meta->make_immutable;
+1;
+
+=head1 COPYRIGHT
+
+Copyright (C) 2017 MetaBrainz Foundation
```

Comment:
You should use your own name here. MetaBrainz is not paying for your work, so you retain your copyright. :)

Reply:
Well I'd like to think I'm part of the team 😸

Reply:
Actually, I did the same as a free contributor in the past. Giving the copyright to the MetaBrainz Foundation is mostly about [_procedural advantages_ [...] _under US copyright law_](https://www.gnu.org/licenses/why-assign.html). Copyright in your own name would be accepted too.

Reply:
Alright. Will do in #588 as well.